packages/flutter/lib/src/rendering/mouse_cursor.dart

```diff
@@ -469,7 +483,9 @@ class SystemMouseCursors {
   /// Corresponds to:
   ///
   ///  * Android: TYPE_WAIT
+  ///  * Windows: IDC_WAIT
   ///  * Web: wait
+  ///  * Linux: wait
   ///
```

Comment:
Can you also add "macOS: [not available]" (and iOS) or something, so that it's clear that it isn't available on macOS/iOS? Or at least a statement that it isn't supported on systems that aren't listed. What happens if I specify `SystemMouseCursor.wait` on a system that doesn't support it?

Reply:
I added the following comment
```
/// If a cursor is not supported on the platform that the application
/// is running on, it falls back to [basic].
```
(Thought I did, thanks for pointing out).